packages/react-graphql-universal-provider/src/network-error-link.ts

```diff
@@ -0,0 +1,21 @@
+import {ApolloLink} from 'apollo-link';
+import {ErrorLink} from 'apollo-link-error';
+import {ServerParseError} from 'apollo-link-http-common';
+
+export function createNetworkErrorLink() {
+  // eslint-disable-next-line no-process-env
+  if (process.env.NODE_ENV === 'development') {
```

Comment:
not sure if this needed to be bind to development. it is this way in web and maybe we are trying to avoid surfacing something?

Reply:
In theory they probably want to build a custom error link for production, either error seems not the most useful thing to surface to users in prod. 

That said, I'd rather we surface the more useful error in prod just because that's where these usually happen since we do auth as infrastructure. At the very least I'd aways log the output so it's more debugabble via splunk  / bugsnag / whatever.

Reply:
I am doing another test to see if the Network error show up in splunk. 
If that work I think I am going to remove the process.env.NODE_ENV === 'development' check then.